lib/apis/v2/node.go

```diff
@@ -41,6 +41,17 @@ type Node struct {
 type NodeSpec struct {
 	// BGP configuration for this node.
 	BGP *NodeBGPSpec `json:"bgp,omitempty" validate:"omitempty"`
+
```

Comment:
@heschlie could you update or add a test here: https://github.com/projectcalico/libcalico-go/blob/master/lib/clientv2/node_e2e_test.go

Reply:
Thought I had moved the old test over, will get it added.

Reply:
Ah I did move over the OrchRef additions to the NodeSpecs in the top of that test. Are the CRUD tests for Node not sufficient coverage of the OrchRefs? I can add some targeted tests if we don't think so.

Reply:
ping @caseydavenport

Reply:
Ah, somehow I didn't see that before.

Yes, that's fine.